idseq_dag/util/m8.py

```diff
@@ -67,7 +71,7 @@
 MIN_CONTIG_SIZE = 4
 
 
-def parse_tsv(path, schema, expect_headers=False, raw_lines=False):
+def parse_tsv(path, schema, expect_headers=False, raw_lines=False, min_alignment_length=0):
```

Comment:
What is the purpose of adding the `min_alignment_length` kwarg here? It doesn't seem to be used within.

Reply:
This was a hold-over from initial work to attempt the "TODO:  Deprecate this iterate_m8() function...". It is no longer relevant, as I took a different approach for the e-value filter. Will remove!